node/src/main/kotlin/net/corda/node/services/statemachine/transitions/TopLevelTransition.kt

```diff
@@ -63,11 +68,11 @@ class TopLevelTransition(
                 is Event.OvernightObservation -> overnightObservationTransition()
                 is Event.WakeUpFromSleep -> wakeUpFromSleepTransition()
                 is Event.Pause -> pausedFlowTransition()
-                is Event.TerminateSessions -> terminateSessionsTransition(event)
             }
         } catch (t: Throwable) {
             // All errors coming from the transition should be sent back to the flow
             // Letting the flow re-enter standard error handling
+            log.warn("Error occurred while creating transition for event: $event", t)
```

Comment:
should we make this error-level? I assume these are never going to be recoverable, right?

Reply:
It _could_ technically recover dependening on the exception thrown. Since it re-enters user code, i think a warn is ok. @rick-r3 worth turning this into an error?

Reply:
Yeah I'd say lets make this error, it's definitely not gone correctly if it's got here.
```suggestion
            log.error("Error occurred while creating transition for event: $event", t)
```

Reply:
hmm, actually yes I might have been wrong..

For example, transient database errors are going to flow through this before being retried, right @lankydan ?

Reply:
No not in this case. This only catches errors when creating the transition. Most of the time this would end the flow but it could technically retry if the exception thrown was a retryable one. But, db exception etc wont come from here. So most of the time these errors will not retry.

Reply:
right, had the action executor in mind - should be fine